accounts-service/com.endlessm.ParentalControls.AppFilter.xml

```diff
@@ -31,5 +31,33 @@
       <annotation name="org.freedesktop.Accounts.DefaultValue"
         value="(false, [])"/>
     </property>
+
+    <!--
+      oars-filter:
+
+      A filter for which applications the user can see in app lists and install.
+      This is intended to be set by administrators and read by users, rather
+      than being editable by the user themselves.
+
+      It’s a two-tuple of the rating type, and a dictionary of rating sections
+      and values. The type gives the rating scheme in use — currently only
+      `oars-1.0` is supported.
```

Comment:
oars-1.1 is available in the wild (I'd say that about 50% of flathub apps use it) and we'll get it in our version of the App Center with our next rebase as it was added in 3.27.3. It's obviously OK to use 1.0 (1.1 is a strict superset) but wanted to give some context.

Reply:
I cannot find a single specification for `oars-1.1`, or reference to what it contains or how it differs from `oars-1.0`. Hence I didn’t know it existed. :-( Do you know of something to point me towards?

Reply:
(But yes, we should definitely support `oars-1.1`.)

Reply:
Followed up at https://github.com/endlessm/eos-parental-controls/pull/5